src/Orchard.Web/Modules/Orchard.OpenId/ViewModels/ErrorViewModel.cs

```diff
@@ -0,0 +1,17 @@
+﻿using System;
+using System.Collections.Generic;
+using System.ComponentModel.DataAnnotations;
+using System.Linq;
+using System.Threading.Tasks;
+
+namespace Orchard.OpenId.ViewModels
+{
+    public class ErrorViewModel
+    {
+        [Display(Name = "Error")]
+        public string Error { get; set; }
+
+        [Display(Name = "Description")]
+        public string ErrorDescription { get; set; }
```

Comment:
It needs to be a `LocalizedString`. Also I didn't see any usage in the views, expected? Or is it just returned in response payloads?